simulator/func_sim/alu.h

```diff
@@ -222,6 +222,9 @@ struct ALU
     // Bit manipulations
     template<typename I> static void sbext( I* instr) { instr->v_dst = 1 & ( instr->v_src1 >> shamt_v_src2<typename I::RegisterUInt>( instr)); }
 
+	// Bit manipulations
+	template<typename I> static void sbinv(I* instr) {instr->v_dst = 1 & (instr->v_src1 >> shamt_v_src2<typename I::RegisterUInt>(instr));}
```

Comment:
What is the difference to `sbext`?

Reply:
I forgotten to upload changes of the file, sorry